app/src/main/res/layout/search_custom_view.xml

```diff
@@ -21,6 +21,7 @@
         android:drawablePadding="@dimen/dimen_6_dp"
         android:drawableTint="@color/neutral_05_light_grey"
         android:gravity="center_vertical"
+        android:singleLine="true"
```

Comment:
`singleLine` is deprecated attribute. Here is what documentation says:

> **@ deprecated** This attribute is deprecated. Use `maxLines` instead to change the layout of a static text, and use the `textMultiLine` flag in the inputType attribute instead for editable text views (if both singleLine and inputType are supplied, the inputType flags will override the value of singleLine).

Reply:
done